applications/API/endpoints.py

```
@@ -0,0 +1,122 @@
+from rest_framework.generics import ListAPIView, RetrieveUpdateDestroyAPIView
```

Pylint (3.3.7)

[warning] Unknown option value for '--disable', expected a valid pylint message and got 'long-suffix'

(W0012)